dfdaemon/config/config.go

```diff
@@ -114,6 +114,11 @@ type Properties struct {
 	LocalIP    string          `yaml:"localIP" json:"localIP"`
 	PeerPort   int             `yaml:"peerPort" json:"peerPort"`
 	StreamMode bool            `yaml:"streamMode" json:"streamMode"`
+
+	ProtocolConf []ProtocolConfig `yaml:"protocolConf" json:"protocolConf"`
+	// DefaultPattern is the default pattern.
+	DefaultPattern string          `yaml:"defaultPattern" json:"defaultPattern"`
```

Comment:
`Defaultxxx` is odd to me. Default values are normally defined by the system itself.

This is a configuration definition, in another word, this is user facing. I'd suggest calling it as just `Pattern` or as @inoc603 mentioned `Mode`.